### DISCUSSION TITLE
Any way to hide the default gray image while it's loading/fetching the image?

### DISCUSSION THREAD

**Discussion by johhansantana** [duguyihou/react-native-turbo-image#381] [Q&A]
if I use the TurboImage component to fetch an URI it will place a gray placeholder while it loads. Is there a way to disable this as I don't want to add a `placeholder` **_blurhash_** for each image. Or is there any recommended way of achieving this?

thanks

**Comment by johhansantana** [comment]
nevermind. while playing around with the props, found out if I add the `indicator` with a style color it will display a small loading which is fine for me. I think this might be a bug if I don't add the `indicator` prop it will fill the width/height of the TurboImage component with a dark gray color.

**Reply by duguyihou** [reply]
hey @johhansantana , can you share a reproducer? I want to look into it.
